src/tests/networking/backend.test.ts

```diff
@@ -29,6 +28,89 @@ afterEach(() => {
   server.close();
 });
 
+describe("httpConfig is setup correctly", () => {
+  function setCustomerInfoResponse(httpResponse: HttpResponse) {
+    server.use(
+      http.get("http://localhost:8000/v1/subscribers/someAppUserId", () => {
+        return httpResponse;
+      }),
+    );
+  }
+
+  test("credentials is omit by default", async () => {
+    setCustomerInfoResponse(
+      HttpResponse.json(customerInfoResponse, { status: 200 }),
+    );
+
+    let requestPerformed: Request | undefined;
+    server.events.on("request:start", (req) => {
+      requestPerformed = req.request;
+    });
+    await backend.getCustomerInfo("someAppUserId");
+    expect(requestPerformed).not.toBeNull();
+    expect(requestPerformed?.credentials).toEqual("omit");
+  });
+
+  test("credentials is include if passing httpConfig parameter", async () => {
```

Comment:
```suggestion
  test("credentials are included if passing httpConfig parameter", async () => {
```

Reply:
Actually this is correct but I should likely put it between quotes :P. Same for the other one